src/main/webapp/js/analytics.js

```diff
@@ -16,51 +16,106 @@
 
+  handleStoreChart(analytics);
+  handleCategoryChart(analytics);
+}
+
+/**
+ * Populates DataTable with store data and passes that to draw method.
+ * @param {Array} analytics First entry is store data, second is category.
+ */
+function handleStoreChart(analytics) {
   storeData = new google.visualization.DataTable();
 
   storeData.addColumn('string', 'Store');
   storeData.addColumn('number', 'Total');
 
-  for (const [store, total] of Object.entries(analytics)) {
+  for (const [store, total] of Object.entries(analytics.storeAnalytics)) {
     storeData.addRow([capitalizeFirstLetters(store), total]);
   }
 
-  drawStoresChart(storeData);
+  drawStoreChart(storeData);
+}
+
+/**
+ * Populates DataTable with category data and passes that to draw method.
+ * @param {Array} analytics First entry is store data, second is category.
+ */
+function handleCategoryChart(analytics) {
+  categoryData = new google.visualization.DataTable();
+
+  categoryData.addColumn('string', 'Category');
+  categoryData.addColumn('number', 'Total');
+
+  for (const [store, total] of Object.entries(analytics.categoryAnalytics)) {
+    categoryData.addRow([capitalizeFirstLetters(store), total]);
+  }
+
+  drawCategoryChart(categoryData);
 }
 
 /**
- * Intializes and draws stores chart onto DOM.
+ * Intializes and draws store chart onto DOM.
  * @param {DataTable} data Table with stores and their totals.
  */
-function drawStoresChart(data) {
+function drawStoreChart(data) {
+  // Get parent div dimensions to make chart fit the appropriate space.
   const chartWidth =
-      document.getElementById('stores-chart').getBoundingClientRect().width;
+      document.getElementById('store-chart').getBoundingClientRect().width;
   const chartHeight =
-      document.getElementById('stores-chart').getBoundingClientRect().height;
+      document.getElementById('store-chart').getBoundingClientRect().height;
 
   const options = {
     title: 'Spending habits by store',
     pieHole: 0.3,
     width: chartWidth,
     legend: {alignment: 'center'},
     chartArea: {width: chartWidth, height: chartHeight * 0.6},
+    sliceVisibilityThreshold: .03,
+  };
+
+  const chart =
+      new google.visualization.PieChart(document.getElementById('store-chart'));
+  chart.draw(data, options);
+}
+
+/**
+ * Intializes and draws category chart onto DOM.
+ * @param {DataTable} data Table with categories and their totals.
+ */
+function drawCategoryChart(data) {
+  // Get parent div dimensions to make chart fit the appropriate space.
+  const chartWidth =
+      document.getElementById('category-chart').getBoundingClientRect().width;
+  const chartHeight =
+      document.getElementById('category-chart').getBoundingClientRect().height;
+
+  const options = {
+    title: 'Spending habits by category',
+    pieHole: 0.3,
+    width: chartWidth,
+    legend: {alignment: 'center'},
+    chartArea: {width: chartWidth, height: chartHeight * 0.6},
+    sliceVisibilityThreshold: .03,
```

Comment:
I'm not sure that using a pie chart would accurately convey category data. Wouldn't having 1 receipt with 2 categories show 50% spent on each? This can be further discussed, but out of the [Chart Gallery](https://developers.google.com/chart/interactive/docs/gallery) I think either the column chart or bar chart would better represent spending.

Reply:
I agree with this. If we *really* wanted to use a pie chart here, we could pick some category (say, the first one) for each receipt to exclusively fall under, but I feel like the selection would end up being arbitrary.